src/ontology/uberon-edit.obo

```diff
@@ -10024,6 +10024,7 @@ xref: Wikipedia:Gonad
 xref: XAO:0003146
 xref: ZFA:0000413
 is_a: UBERON:0003133 ! reproductive organ
+is_a: UBERON:3011048 ! genital system
```

Comment:
I think part_of would be better.

Reply:
Also for most of all other children.  Same scope for refactoring to clean up too given hidden GCIs.